src/Fixers.php

```diff
@@ -122,93 +196,198 @@ class Fixers
      */
     public static $risky = array(
         'ereg_to_preg',
-        'php4_constructor',
+        'no_php4_constructor',
         'php_unit_construct',
+        'php_unit_dedicate_assert',
         'php_unit_strict',
         'psr0',
         'psr4',
-        'strict',
+        'strict_comparison',
         'strict_param',
     );
 
 
     /**
      * @var string[]
      */
+    public static $aliases = array(
+        'operators_spaces' => 'binary_operator_spaces',
+        'line_after_namespace' => 'blank_line_after_namespace',
+        'blankline_after_open_tag' => 'blank_line_after_opening_tag',
+        'return' => 'blank_line_before_return',
+        'spaces_cast' => 'cast_spaces',
+        'short_tag' => 'full_opening_tag',
+        'newline_after_open_tag' => 'linebreak_after_opening_tag',
+        'alias_functions' => 'no_alias_functions',
+        'join_function' => 'no_alias_functions',
```

Comment:
> PHP-CS-Fixer removed the join_function fixer in 2.x, and replaced it with alias_functions, and then renamed that one to no_alias_functions.

Was `alias_functions` name pushed on a stable version? If yes, it should be kept on the aliases section.